app/assets/stylesheets/welcome.scss

```diff
@@ -79,14 +81,6 @@ ul{
   }
 }
 
-@-moz-document url-prefix() {
-  a {
-    &:active, &:focus{
-      outline: none;
-      border:  none;
-    }
-  }
-  .btn:focus, .btn.focus, .btn:active:focus, .btn.focus:active, .btn.active:focus, .btn.active.focus{
-    outline: none;border:  none; box-shadow: none;
-  }
+:focus {
```

Comment:
What do you think about using the same styles we're using in CONSUL (check commits [35a45837f](https://github.com/consul/consul/pull/4679/commits/35a45837f) and [bd135966c](https://github.com/consul/consul/pull/4679/commits/bd135966ca) for explanations)? :thinking:

```scss
a,
button,
[type="button"],
[type="submit"] {
  &:focus {
    outline: $outline-focus;
  }

  &:focus-visible {
    outline: $outline-focus;
  }

  &:focus:not(:focus-visible) {
    outline: none;
  }

  &:active,
  &:focus:active {
    outline: $outline-focus;
  }
}
```

These styles only apply to buttons and links. The reason why they weren't applied to form inputs is I found the outline a bit too aggressive :thinking:. Besides, the outline didn't work well for radio buttons and check boxes on some browsers (although I think in rockandror.com we don't have those controls).

For form inputs I think the most common pattern is to change the `border` property on focus instead of the `outline` property :thinking:. In the past I've used a different background color instead of an outline when focusing on form inputs, and I seem to recall it also worked well, but it's probably not that common.